test/benchmark/005-authority-delegation-unsecure.js

```diff
@@ -1,325 +1,366 @@
-              $happn.data.set('/data/forbidden', {
-                test: 'data'
-              }, callback);
-            },
-            otherMethod: function($happn, callback) {
-              callback(new Error('unexpected success'))
+          },
+          'y-service-name': {
+            instance: {
+              method1: function(callback) {
+                callback(null, 'y-service-name/method1 ok');
+              },
+              allowedMethodNotData: function($happn, callback) {
+                $happn.data.set(
+                  '/data/forbidden',
+                  {
+                    test: 'data'
+                  },
+                  callback
+                );
+              },
+              otherMethod: function($happn, callback) {
+                callback(new Error('unexpected success'));
+              }
             }
           }
-        }
-      },
-      components: {
-        'service-name': {
-          security: {
-            authorityDelegationOn: true
-          }
         },
-        'x-service-name': {
-          security: {
-            authorityDelegationOn: true
-          }
-        },
-        'y-service-name': {
-          security: {
-            authorityDelegationOn: false
-          }
-        }
-      }
-    }).then(function(_mesh) {
-      secureMesh = _mesh;
-      done();
-    }).catch(done);
-  });
-
-  before('setup secureMesh user', function(done){
-    var theGroup = {
-      name: 'group',
-      permissions: {
-        methods: {
-          '/secureMesh/service-name/method1': {
-            authorized: true
-          },
-          '/secureMesh/service-name/allowedMethodNotData': {
-            authorized: true
-          },
-          '/secureMesh/x-service-name/allowedMethodNotData': {
-            authorized: true
-          },
-          '/secureMesh/y-service-name/allowedMethodNotData': {
-            authorized: true
-          },
-          '/secureMesh/service-name/allowedMethodNotOtherMethod': {
-            authorized: true
+        components: {
+          'service-name': {
+            security: {
+              authorityDelegationOn: true
+            }
           },
-          '/secureMesh/x-service-name/otherMethod': {
-            authorized: true
+          'x-service-name': {
+            security: {
+              authorityDelegationOn: true
+            }
           },
-        },
-        data: {
-          '/data/forbidden': {
-            authorized: false,
-            actions: ['set']
+          'y-service-name': {
+            security: {
+              authorityDelegationOn: false
+            }
           }
         }
-      }
-    };
+      })
+        .then(function(_mesh) {
+          secureMesh = _mesh;
+          done();
+        })
+        .catch(done);
+    });
 
-    var theUser = {
-      username: 'username',
-      password: 'password'
-    };
+    before('setup secureMesh user', function(done) {
+      // var theGroup = {
```

Comment:
please delete dead code or add a comment explaining that it's dead, and why it's being kept